test/extended/operators/server_side_apply_examples.go

```diff
@@ -90,7 +90,7 @@ var _ = g.Describe("[sig-apimachinery]", func() {
 			}
 		})
 
-		g.It("should clear fields when they are no longer being applied in FeatureGates", func() {
+		g.It("should clear fields when they are no longer being applied in FeatureGates [apigroup:config.openshift.io]", func() {
```

Comment:
This looks good, @stbenjam is there a default stablemap impl in https://github.com/openshift-eng/ci-test-mapping that will automatically omit the apigroup tag from the stable test name? Or will this require a custom rename?

Reply:
I don't think so but we should just handle stuff like this https://issues.redhat.com/browse/TRT-1716